### DISCUSSION TITLE
[Question] Mob Fanning

### DISCUSSION THREAD

**Discussion by markvetwow** [azerothcore/azerothcore-wotlk#3906] [General]
Hi! I am noob at programming, my programming skill is basically limited to easy script fixes, so there's a question. I found two patches for mob fanning that wasn't pulled to tc, or just wasn't tested before. Is there any way to implement/convert something like this to AzerothCore?

First file -> [Click](https://gist.github.com/Langerz82/1dbe4acec7fb6bbedf61fb4a44aeece3#file-movementbackandcircle-diff)

Second file that i found on ac-web i guess -> [0185-Core-Movement-somewhat-implement-mob-fanning.txt](https://github.com/azerothcore/azerothcore-wotlk/files/4688885/0185-Core-Movement-somewhat-implement-mob-fanning.txt)

**Comment by FrancescoBorzi** [comment]
moved to forum

**Comment by ghost** [comment]
There's indeed a lot of work to do regarding mob behaviour and pathfinding. Nice links though, there's a lot of logic that can be re-used from these.

**Comment by BarbzYHOOL** [comment]
i already got the second file saved long ago, people checked it and said it's far from enough (@Nefertumm  looked at it i think)

first one, never seen it
